src/components/PressableWithSecondaryInteraction/index.js

```diff
@@ -50,21 +55,22 @@ class PressableWithSecondaryInteraction extends Component {
 
     render() {
         const defaultPressableProps = _.omit(this.props, ['onSecondaryInteraction', 'children', 'onLongPress']);
+        const executeSecondaryInteraction = (e) => {
```

Comment:
One last thing, this should be a class method we can call using `this.executeSecondaryInteraction`